### ISSUE TITLE
testnet faucet sites invalid

### ISSUE THREAD

**Issue by systemsplanet** [bitcoinj/bitcoinj.github.io#33]
[testing page](https://github.com/bitcoinj/bitcoinj.github.io/blob/master/testing.md)
```
Most usefully, coins on the testnet have no value and can be obtained for free from testnet faucet sites like http://faucet.xeno-genesis.com/ or http://tpfaucet.appspot.com/.
```

Either those sites are no longer available or blocked by corp firewall

Are there other sites?
Is there doc on how to create a stand-alone test setup for 
- localhost
- dev sever pool

**Comment by gdassori** [comment]
A recent faucet is https://testnet-faucet.mempool.co

**Comment by schildbach** [comment]
Fixed via fa5e3291c3b613b1d88703f107595fe2f1d14c47.
